ui/src/app/views/workflow/show/admin/delete-modal/delete-modal.html

```diff
@@ -0,0 +1,49 @@
+<ng-template let-context let-modal="modal" #workflowDeleteModal>
+    <div class="header">
+        Deleting workflow {{workflow.name}}
+    </div>
+    <div class="content scrolling" [loading]="loading">
+        <div class="ui grid">
+            <div class="row">
+                <div class="sixteen wide column">
+                    <h3>Do you want to delete the workflow and clean its dependencies ?</h3>
```

Comment:
Can we disable the form "with dependencies" when no dependencies are found ?

Reply:
done